code/game/machinery/portable_turret.dm

```diff
@@ -733,35 +622,30 @@ Status: []<BR>"},
 	if (!raised) // the turret has to be raised in order to fire - makes sense, right?
 		return
 
-
-	// any emagged turrets will shoot extremely fast! This not only is deadly, but drains a lot power!
-
-	if(iconholder)
+	if(!emagged)
 		icon_state = "[lasercolor]target_prism"
 	else
 		icon_state = "[lasercolor]orange_target_prism"
 
+	use_power(reqpower)
+
+//		 //Shooting Code:
```

Comment:
I tried that first, most of Fire() is set up expecting a mob/living user and I didn't want to mess with it too much